WindowsServerDocs/administration/OpenSSH/OpenSSH_KeyManagement.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: OpenSSH key management for Windows
 description: OpenSSH Server key management for Windows using the Windows tools or PowerShell.
-ms.date: 09/27/2018
+ms.date: 09/09/2020
```

Comment:
Not yet merged, so I propose a date this month, just to be on the safe side:
```suggestion
ms.date: 11/11/2020
```

Reply:
Should it be`11/04/2020`?

Reply:
Well, sure, but perhaps I was a bit pessimistic about today being the day this gets merged, so I thought the suggested date would be much closer to the actual merge date. Also, I like to see dates presented in an unambiguous form, regardless of month/date (perhaps a bit overzealous on my part).

Reply:
Tbh, I don't know if `ms.date` is when the document is written or merge, but 11/04/2020 (no comment regarding the formatting) is a known.